### PR TITLE
Corrections for Chapter 1 + 2 of Episode 10 

### PR DESCRIPTION
**Pull request by Ryman**
I've perhaps been too eager with `s/web audio/Web Audio`, I can undo that if you'd prefer, but I had noticed you'd been specific with `WebGL` so I followed precedence.

I'm unsure about the 3rd commit, the text matched what was said but it was really hard to follow if you imagine reading it without sound. I can drop the commit if you want!

The style guide mentioned a 44 character limit per line, but the majority of lines seemed to be over that already so I was unsure if that was still a requirement. I can try to cut some lines down or reformat if desired.

Also, caption 8 starts with `<head>` which doesn't seem to render on the wistia player but is fine in VLC. Wasn't able to test that locally so I don't have a suggested fix.

### CODE REVIEW

**Comment by nelstrom** [comment]
@Ryman great job, thanks.

I'm cool with `s/web audio/Web Audio` 👍 

I'm going to leave out your [third commit](https://github.com/fromPeerToPeer/captions/pull/79/commits/48183cb7ac2d79a471a9b0ac8e1ddc1ff1fc22ee). The captions should make all that is spoken accessible to someone who can't hear the dialogue, but for the captions to provide extra contextual information is going beyond their brief. I'll try and take your suggestion and do something else with it though. Perhaps we could overlay a 'cheatsheet' ([like this](http://peertopeer.io/images/soledad-penades/cheatsheet-timeout-interval.jpg)) that gives a bit of extra context. That way, the information would be available to viewers whether they are using captions or not.

**Comment by Ryman** [comment]
> Perhaps we could overlay a 'cheatsheet' (like this) that gives a bit of extra context.

That seems like a much better solution! :+1:
